package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "ai-flow-dashboards",
+  "version": "3.0.0.0",
+  "description": "OpenSearch AI Flow Dashboards Plugin",
+  "main": "index.js",
+  "config": {
+    "plugin_version": "3.0.0.0",
+    "plugin_name": "aiFlowDashboards",
+    "plugin_zip_name": "ai-flow-dashboards"
```

Comment:
same question for these plugin/zip names

Reply:
See https://github.com/opensearch-project/opensearch-ai-flow-dashboards/pull/3#discussion_r1316201499